application/values.yaml

```diff
@@ -95,7 +95,7 @@ deployment:
     #   timeFormat: world
 
   # Replicas to be created
-  replicas: 
+  replicas: 1
```

Comment:
This doesnt help and will causes issues when used in conjunction with HorizontalPodAutoscaling 

For example ArgoCD will sync 1 replica, HPA will try scale up and they will flip flop for ever

Reply:
we should add this in the values file; I mean this reasoning

Reply:
also just to point out a deployment that omits the `replicas:` is still perfectly valid, kubernetes defaults to 1 replica